airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/ConfigPersistence.java

```diff
@@ -31,6 +31,8 @@
 
 public interface ConfigPersistence {
 
+  String getAirbyteVersion();
```

Comment:
why do we need to add `getAirbyteVersion` to the config persistence / config repository. could the MigrationHandler just handle this (in other words we just pass the airbyte version into the migration handler constructor) and config persistence doesn't have to worry about it? if it were doing something in config persistence it would be different, but right now it just feels like it is adding noise.

Reply:
I still haven't addressed this issue with the version yet, will do that tomorrow